TestHotel/TestHotel.DataAccess/Repository/Repositories/RoomTypeRepository.cs

```diff
@@ -55,12 +57,14 @@ public async Task<RoomType> GetRoomTypeByIdAsync(RoomTypes roomTypes)
         {
             try
             {
+                _logger.LogInformation("RoomTypeById muvaffaqiyatli topildi");
                 return await _context.RoomTypes
                     .Include(u => u.Room)
                     .FirstOrDefaultAsync(u => u.RoomTypes == roomTypes);
```

Comment:
RoomTypeRepository dagi va IRoomTypeRepository dagi GetRoomTypeById ni RoomTypeId oladigan va 
(u => u.RoomTypeId == roomTypeId) bo'lishi kerak

Reply:
it's out of scope of this ticket. Please, Create a defect ticket for it. @MirazamTUT

Reply:
hop